framework/behaviors/SluggableBehavior.php

```diff
@@ -144,7 +150,9 @@ protected function getValue($event)
                 foreach ((array) $this->attribute as $attribute) {
                     $slugParts[] = ArrayHelper::getValue($this->owner, $attribute);
```

Comment:
Empty check and return could be done right here so there's no need to iterate array one more time.